code/_onclick/hud/radial.dm

```diff
@@ -254,6 +254,9 @@ GLOBAL_LIST_EMPTY(radial_menus)
 
 /datum/radial_menu/proc/hide()
 	if(current_user)
+		for(var/obj/screen/radial/slice/slice_to_remove in elements)
+			slice_to_remove.parent = null
+		close_button.parent = null
```

Comment:
Shouldn't this be part of the Destroy instead? Also the elements themselves are not being deleted

Reply:
yeah probably
also bwuh